src/OrchardCore.Modules/OrchardCore.Media/Processing/MediaImageSharpConfiguration.cs

```diff
@@ -27,17 +27,17 @@ public void Configure(ImageSharpMiddlewareOptions options)
             {
                 // Force some parameters to prevent disk filling.
                 // For more advanced resize parameters the usage of profiles will be necessary.
-                // This can be done with a custom IImageWebProcessor implementation that would 
+                // This can be done with a custom IImageWebProcessor implementation that would
                 // accept profile names.
 
-                validation.Commands.Remove(ResizeWebProcessor.Compand);
-                validation.Commands.Remove(ResizeWebProcessor.Sampler);
-                validation.Commands.Remove(ResizeWebProcessor.Xy);
-                validation.Commands.Remove(ResizeWebProcessor.Anchor);
-                validation.Commands.Remove(BackgroundColorWebProcessor.Color);
-
                 if (validation.Commands.Count > 0)
                 {
+                    validation.Commands.Remove(ResizeWebProcessor.Compand);
+                    validation.Commands.Remove(ResizeWebProcessor.Sampler);
+                    validation.Commands.Remove(ResizeWebProcessor.Xy);
+                    validation.Commands.Remove(ResizeWebProcessor.Anchor);
+                    validation.Commands.Remove(BackgroundColorWebProcessor.Color);
+
                     if (!validation.Commands.ContainsKey(ResizeWebProcessor.Mode))
```

Comment:
This original logic actually makes me wonder, if commands is empty, the mode will never be added - is this a problem?

Reply:
No what you have done here is great.
The commands are just a parse of the query string. (`/media/01.jpg?width=200` with width being a command).
So with the change to ImageSharp it will no-op of there are no commands in the query string. 
Here we just add the resize mode = `max` if there are resize commands, as a default, but only if there is resizing to be done. There won't be any resizing (I think) on the blog theme, but there is on the agency theme.
ImageSharp could resolve the commands later, as they are not used until after the providers are matched, but they don't like my pr for that, so they make the change to no-op instead. Which is cool